submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/SubmissionRepository.kt

```diff
@@ -22,8 +23,9 @@ class SubmissionRepository(
     private val submissionDbMapper: SubmissionDbMapper,
     private var toExtSubmissionMapper: ToExtSubmissionMapper
 ) {
-    fun getByAccNo(accNo: String): Submission = submissionDbMapper.toSubmission(getSubmission(accNo))
+    fun getByAccNo(accNo: String): Submission = getExtByAccNo(accNo).toSimpleSubmission()
 
+    @Deprecated("Extended model has been deprecated")
```

Comment:
I understand the importance of deprecating but, since we're the only ones using this model right now, I think we could just remove it altogether

Reply:
there are still some parts using it :(